lookout/style/format/features.py

```diff
@@ -240,23 +244,54 @@ def feature2index(self) -> Dict[str, int]:
 
     def count_features(self, feature_type: FeatureType) -> int:
         """
-        Returns the number of features belonging to a specific type.
+        Return the number of features belonging to a specific type.
```

Comment:
I think we usually use `s` in the end of the verb. Because we omit `function` word so it is 
> `The function` return_s_ the number of features belonging to a specific type.

Reply:
https://www.python.org/dev/peps/pep-0257/
I didn't see anything in the guide mentioning that we do not use the standard but if we do it'd be interesting to document :)